pkg/sfu/buffer/rtpstats_base.go

```diff
@@ -34,7 +34,7 @@ const (
 	cFirstSnapshotID     = 1
 
 	cFirstPacketTimeAdjustWindow    = 2 * time.Minute
-	cFirstPacketTimeAdjustThreshold = 5 * time.Minute
+	cFirstPacketTimeAdjustThreshold = 1 * time.Minute
```

Comment:
This is used to threshold first packet time adjustment. If the calculated adjustment is more than this, we declare a bad sample and ignore. Even 1 minute is a lot there.

Reply:
should this be even lower than 1 min then? what's the downside of setting it to say.. 15s?